### PR TITLE
move apk installs to the existing...

### PR DESCRIPTION
**Pull request by nickboldt**
move apk installs to the existing section

Change-Id: I859abaa25bb5106cc6651043b364987ac7aa7463
Signed-off-by: nickboldt <nboldt@redhat.com>

### CODE REVIEW

**Comment by nickboldt** [comment]
Replaces #3

**Comment by nickboldt** [comment]
@benoitf can we rebuild 1.12.9 or do we need a new branch for 1.12.9a?

Also could these changes simply be in master branch, then when it's time to cut a new release, we simply branch from master? I'm curious why we're working in branches instead of working in master (as we do for all the other Che projects. 

This approach seems to make it harder to submit PRs - when the target branch doesn't exist, your PR has to involve CREATING a branch as well as just committing to an existing one.

**Comment by benoitf** [comment]
as long it's still 1.12.9 the branch is updated.

**Review by benoitf — COMMENTED** [review]
git is no longer installed ?

**Comment by nickboldt** [comment]
> git is no longer installed ?

See line 30: https://github.com/che-dockerfiles/che-sidecar-go/pull/4/files#diff-3254677a7917c6c01f55212f86c57fbfR30 (git already installed earlier)

**Review by benoitf — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -23,7 +23,7 @@ RUN set -e \
     && \
     apk add --update --no-cache  --virtual .build-deps \
         bash \
-        gcc \
+        gcc g++ \
```
Comment (amisevsk) [inline Dockerfile:26]: I'd keep the format of one package per line, but it's not very important.

**Review by amisevsk — APPROVED** [review]
Note that this PR increases the image size from `935MB` to `1.25GB`

**Pull request merged**